exercises/Perception-Driven_Manipulation/solution_ws/src/collision_avoidance_pick_and_place/src/tasks/pickup_box.cpp

```diff
@@ -31,7 +31,6 @@ void collision_avoidance_pick_and_place::PickAndPlace::pickup_box(std::vector<ge
      * - The WRIST_LINK_NAME field in the "cfg" configuration member contains
      *  the name for the arm's wrist link.
      */
-    move_group_ptr->setEndEffector(cfg.WRIST_LINK_NAME);
```

Comment:
Should this have been deleted?

Reply:
It was not necessary since the manipulator group contains the tcp link